backend/src/email/utils/callback/query.ts

```diff
@@ -185,6 +185,7 @@ export const haltCampaignIfThresholdExceeded = async (
         await EmailBlacklist?.sequelize?.transaction(async (transaction) => {
           const [numUpdated] = await Campaign.update(
             { halted: true },
+            // this allows for manual override of halting: if halted is set to null, campaign will not be halted
```

Comment:
A better place to put this comment is in the model rather than here

Reply:
ok have put it there